app/screens/planner/planner-screen.tsx

```diff
@@ -292,6 +303,13 @@ export const PlannerScreen = observer(function PlannerScreen({ navigation }: Pla
           value={formattedDate}
           style={{ marginBottom: spacing[5] }}
           onPress={() => setDatePickerVisibility(true)}
+          sideSection={
+            formattedDate !== translate("plan.now") && (
+              <TouchableOpacity hitSlop={spacing[4]} onPress={() => onDateChange(new Date())}>
```

Comment:
add `accessibilityLabel` for the icon, instructing it's resetting the date to the current date